packages/blade/src/components/Card/_decisions/decisions.md

```diff
@@ -0,0 +1,512 @@
+      title="Card Footer Title" 
+      subtitle="Card footer subtitle" 
+    />
+    <Card.FooterTrailing 
+      actions={{
+        primaryAction: { text: 'Know more', onClick: () => {} },
+        secondaryAction: { text: 'Read Docs', onClick: () => {} },
+      }}
+    />
+  </Card.Footer>
+</Card>
+```
+
+
+### `Card` API
+
+
+| Prop              | Type   | Default | Description                                                                            | Required |
+| ----------------- | ------ | ------- | -------------------------------------------------------------------------------------- | -------- |
+| `surfaceLevel` | `2, 3` | `2`     | Surface level of the card background color, use this based on where the card is placed |          |
+
+
+### `Card.HeaderLeading` API
+
+
+| Prop             | Type              | Default     | Description                                              | Required |
+| ---------------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| `title`          | `string`          | `undefined` | Title of the Card                                        | ✅        |
+| `subtitle`       | `string`          | `undefined` | Subtitle of the Card                                     |          |
+| `prefix`    | `React.ReactNode` | `undefined` | Prefix element placed before title text (restricted to: Icon)       |          |
+| `suffix`    | `React.ReactNode` | `undefined` | Suffix element placed after title text (restricted to: Counter)     |          |
+
+### `Card.HeaderTrailing` API
+
+| Prop             | Type              | Default     | Description                                              | Required |
+| ---------------- | ----------------- | ----------- | -------------------------------------------------------- | -------- |
+| `visual` | `React.ReactNode` | `undefined` | Trailing visual element placed on right side of the card (restricted to: Badge, Text, Link, IconButton) |          |
+
+### `Card.FooterLeading` API
+
+| Prop       | Type                                     | Default     | Description                                | Required |
+| ---------- | ---------------------------------------- | ----------- | ------------------------------------------ | -------- |
+| `title`    | `string`                                 | `undefined` | Title of the Card                          | ✅        |
+| `subtitle` | `string`                                 | `undefined` | Subtitle of the Card                       |          |
+
+### `Card.FooterTrailing` API
+
+| Prop       | Type                                     | Default     | Description                                | Required |
+| ---------- | ---------------------------------------- | ----------- | ------------------------------------------ | -------- |
+| `actions`  | `{ primary: Action, secondary: Action }` | `undefined` | Renders a primary/secondary action buttons |          |
+
```

Comment:
What about `Card.Header`, `Card.Footer` and `Card.Body`?

Reply:
Do these also accept any JSX as children? For customizing header, footer etc.?

Reply:
Sorry could not understand, can you elaborate?

Reply:
I mean would users be able to pass any JSX inside the subcomponents? Example:

```tsx
<Card.Header>
  <MyJSX />
</Card.Header>
```

Reply:
Yeah they can. 
Technically we can enforce this on the runtime, but don't think it's necessary cuz that's on them if they want to opt out of the header and add their own stuff.